ci/docker/debug-cpu.yaml

```diff
@@ -9,36 +9,22 @@
 #
 
 spack:
-  specs:
-    - dla-future@develop build_type=Debug +miniapps +ci-test ^openblas ^mpich
+  include:
+  - common.yaml
+
   view: false
   concretizer:
     unify:
       true
```

Comment:
Note that this is common to all the environments, but it seems like spack allow having these in the common config file, so I've left this duplicated.

Reply:
You mean **doesn't** allow?

Reply:
> Note that this is common to all the environments, but it seems like spack (edit:) *does not* allow having these in the common config file, so I've left this duplicated.